### PR TITLE
📝 fix typo in date.md

### PR DESCRIPTION
**Pull request by bennettp123**
fix typo in the doco for date arbitraries at https://fast-check.dev/docs/core-blocks/arbitraries/primitives/date/

**_Category:_**

- [ ] ✨ Introduce new features
- [x] 📝 Add or update documentation
- [ ] ✅ Add or update tests
- [ ] 🐛 Fix a bug
- [ ] 🏷️ Add or update types
- [ ] ⚡️ Improve performance
- [ ] _Other(s):_ ...
  
  

**_Potential impacts:_**

- [ ] Generated values
- [ ] Shrink values
- [ ] Performance
- [ ] Typings
- [ ] _Other(s):_ ...

### CODE REVIEW

File: website/docs/core-blocks/arbitraries/primitives/date.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ slug: /core-blocks/arbitraries/primitives/date/
 
 # Date
 
-Generate boolean values.
+Generate Date values.
```
Comment (dubzzz) [inline website/docs/core-blocks/arbitraries/primitives/date.md:7]:
```suggestion
Generate date values.
```

**Review by dubzzz — COMMENTED** [review]
Thanks a lot for the contribution. As other "Generate xxx values" use lowercase, I'd do the same on this one.

Additionally, you'll have to run `yarn && yarn version:bump` on this PR (mark fast-check has patched and all others as declined)

**Comment by bennettp123** [comment]
> Thanks a lot for the contribution. As other "Generate xxx values" use lowercase, I'd do the same on this one.
> 
> Additionally, you'll have to run `yarn && yarn version:bump` on this PR (mark fast-check has patched and all others as declined)

Thanks—I've updated the PR to include your suggestion to use lowercase :+1:

I've also added a version bump, but I'm not sure I've done it correctly—`yarn && yarn version:bump` doesn't seem to do anything at all, so I committed the results of the command below:

```sh
yarn \
  && yarn workspaces foreach --all --exclude fast-check version decline --deferred \
  && yarn workspace fast-check version patch --deferred
```

**Review by dubzzz — APPROVED** [review]
Perfect

**Comment by dubzzz** [comment]
Thanks a lot for this contribution.

**Pull request merged**